src/MainWindow.cpp

```diff
@@ -89,61 +89,47 @@ MainWindow::MainWindow(BRect frame) : BWindow(frame, "", B_DOCUMENT_WINDOW, 0)
 	BMenuBar* bar = new BMenuBar("keybar");
 
 	BMenu* menu = new BMenu(B_TRANSLATE("Program"));
-
-	temp = B_TRANSLATE("Report a bug…");
-	menu->AddItem(new BMenuItem(temp.String(), new BMessage(M_REPORT_BUG)));
-	temp = B_TRANSLATE("Settings…");
-	menu->AddItem(new BMenuItem(temp.String(), new BMessage(M_SHOW_OPTIONS_WINDOW), ','));
-
+	menu->AddItem(new BMenuItem(B_TRANSLATE("Report a bug…"), new BMessage(M_REPORT_BUG)));
+	menu->AddItem(new BMenuItem(B_TRANSLATE("Settings…"), new BMessage(M_SHOW_OPTIONS_WINDOW), ','));
+	menu->AddSeparatorItem();
+	menu->AddItem(new BMenuItem(B_TRANSLATE("About CapitalBe…"), new BMessage(M_SHOW_ABOUT)));
```

Comment:
The About strictly doen't need a "…". Invoking the menu item does what the label says, showing the AboutWindow.